src/main/java/net/pms/encoders/YoutubeDl.java

```diff
@@ -66,21 +93,29 @@ public boolean isTimeSeekable() {
 
 	@Override
 	public synchronized ProcessWrapper launchTranscode(
-		StoreItem resource,
-		MediaInfo media,
-		OutputParams params
-	) throws IOException {
+			StoreItem resource,
+			MediaInfo media,
+			OutputParams params) throws IOException {
 		params.setMinBufferSize(params.getMinFileSize());
 		params.setSecondReadMinSize(100000);
 		// Use device-specific conf
 		UmsConfiguration configuration = params.getMediaRenderer().getUmsConfiguration();
 		String filename = resource.getFileName();
+
+		String videoId = extractVideoId(resource.getMediaURL());
+		LOGGER.debug("Launching downloader for video ID: {}", videoId);
+
 		setAudioAndSubs(resource, params);
 
 		// Build the command line
 		List<String> cmdList = new ArrayList<>();
 
-		cmdList.add(configuration.getYoutubeDlPath());
+		String downloaderPath = configuration.getYoutubeDlPath();
+		if (!new File(downloaderPath).exists()) {
+			downloaderPath = "/usr/bin/yt-dlp"; // fallback path
```

Comment:
It would be good to make this work across platforms too, if you dig into that `getYoutubeDlPath` you'll see how we usually do that.